File: app/src/main/java/mb00/android/codehub/api/model/Author.java
package mb00.android.codehub.api.model;

import com.google.gson.annotations.SerializedName;


public class Author {

    @SerializedName("name")
    private String name;
    @SerializedName("date")
    private String date;
    @SerializedName("login")
    private String login;
    @SerializedName("avatar_url")
    private String avatarUrl;

    // Mapping for nested JSON when getRepoCommits() is called in GitHubService.java
    public String getName() {
        return name;
    }

    // Mapping for nested JSON when getRepoCommits() is called in GitHubService.java
    public String getDate() {
        return date;
    }

    // Mapping for nested JSON when getRepoReleases() is called in GitHubService.java
    public String getLogin() {
        return login;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

}

File: app/src/main/java/mb00/android/codehub/api/model/Author.kt
package mb00.android.codehub.api.model

import com.google.gson.annotations.SerializedName


data class Author(
        @SerializedName("name") val name: String?,
        @SerializedName("date") val date: String?,
        @SerializedName("login") val login: String?,
        @SerializedName("avatar_url") val avatarUrl: String?
)